livre/eq/Channel.cpp

```diff
@@ -582,16 +624,16 @@ bool Channel::configExit()
 }
 
 void Channel::frameStart( const eq::uint128_t& frameID,
-                          const uint32_t frameNumber )
+                          const uint32_t frameCounter )
 {
-    _impl->_drawRange = eq::Range::ALL;
-    eq::Channel::frameStart( frameID, frameNumber );
+    _impl->frameStart( frameCounter );
+    eq::Channel::frameStart( frameID, frameCounter );
 }
 
 void Channel::frameDraw( const lunchbox::uint128_t& frameId )
 {
     eq::Channel::frameDraw( frameId );
-    _impl->frameDraw( frameId );
+    _impl->frameDraw();
```

Comment:
That's not my point. What I mean is that you have implementation funcions for both frameStart and frameDraw but inside Channel you are not consistently calling these function after or before the eq::Channel versions. My suggestions is that is you cannot move both to be after or before, then rename the internal ones to prefix post or pre according to the moment in which they are calling in relation to the eq::Channel function.